src/code/z_actor.c

```diff
@@ -2821,11 +2846,11 @@ Actor* Actor_Spawn(ActorContext* actorCtx, PlayState* play, s16 actorId, f32 pos
             overlayEntry->numLoaded = 0;
         }
 
-        actorInit = (void*)(uintptr_t)((overlayEntry->initInfo != NULL)
-                                           ? (void*)((uintptr_t)overlayEntry->initInfo -
-                                                     (intptr_t)((uintptr_t)overlayEntry->vramStart -
-                                                                (uintptr_t)overlayEntry->loadedRamAddr))
-                                           : NULL);
+        actorInit = (void*)(uintptr_t)(
```

Comment:
this formatting change is backwards from main (how it is in main is properly formatted afaict)

may be a clang-format version issue, are you using 14?

Reply:
I think the format version tied to vscode (using the clang format plugin) is not 14, even though i do have it installed. using the python script fixed it